ckcp/gitops.sh

```diff
@@ -318,7 +312,78 @@ install_triggers(){
   echo
 }
 
+install_triggers_crds(){
+  APP="triggers-crds"
+  export KUBECONFIG="$KUBECONFIG_CLUSTER"
 
+  #############################################################################
+  # Install triggers CRDs
+  #############################################################################
+  echo -n "  - $APP application: "
+  if ! oc get apps -n openshift-gitops "$APP" >/dev/null 2>&1; then
+    oc apply -f "$GITOPS_DIR/$APP.yaml" --wait >/dev/null
+  fi
+  argocd app wait $APP >/dev/null
+
+
+
+  echo "OK"
+}
+
+install_triggers_interceptors(){
+  APP="triggers-interceptors"
+  export KUBECONFIG="$KUBECONFIG_CLUSTER"
+
+  #############################################################################
+  # Create kcp-kubeconfig secrets for event listener and interceptors so that they can talk to KCP
+  #############################################################################
+  if ! KUBECONFIG=$KUBECONFIG_KCP oc get secret kcp-kubeconfig >/dev/null 2>&1; then
+    echo -n "."
+    KUBECONFIG=$KUBECONFIG_KCP oc create secret generic kcp-kubeconfig --from-file "$KUBECONFIG_KCP" >/dev/null
+  fi
+  if ! KUBECONFIG=$KUBECONFIG_KCP oc get secret kcp-kubeconfig -n tekton-pipelines >/dev/null 2>&1; then
+    echo -n "."
```

Comment:
makes dot on wrong place -> `..  - triggers-interceptors application: OK`. I would just remove it.